scripts/Mekanism.zs

```diff
@@ -269,4 +268,7 @@ for input, output in itemsToUnify {
 	rh(<mekanism:oreblock:2>);
 	rh(<mekanism:walkietalkie>);
 
-print("--- Mekanism.zs initialized ---");
\ No newline at end of file
+# Tier applying (for AE extended recipe)
```

Comment:
Please remove this, as it will void the contents of the machine you're upgrading.

Reply:
This is only problem? I can write function that saves all NBT tags and just changes Tier